packages/fast-usdc/src/exos/liquidity-pool.js

```diff
@@ -63,14 +82,12 @@ export const prepareLiquidityPoolKit = (zone, zcf, brands, tools) => {
      */
     (shareMint, node) => {
       const { brand: PoolShares } = shareMint.getIssuerRecord();
-      const { USDC } = brands;
       const proposalShapes = makeProposalShapes({ USDC, PoolShares });
-      const dust = AmountMath.make(USDC, 1n);
-      const shareWorth = makeParity(dust, PoolShares);
+      const shareWorth = makeParity(makeDust(USDC), PoolShares);
       const { zcfSeat: poolSeat } = zcf.makeEmptySeatKit();
       const shareWorthRecorderKit = tools.makeRecorderKit(
         node,
-        /** @type {TypedPattern<ShareWorth>} */ (RatioShape),
+        /** @type {TypedPattern<Ratio>} */ (RatioShape),
```

Comment:
please annotate `RatioShape` instead

Reply:
`Ratio` is defined in `@agoric/zoe`. Adding a (dev)dependency from ERTP to Zoe seems odd.

Does annotating `shareWorthRecorderKit` work for you?